### PR TITLE
Add content for slide deck 4

### PR DESCRIPTION
**Pull request by juliasilge**

### CODE REVIEW

File: slides/04-monitor.qmd [hunk 1]
```diff
@@ -109,13 +171,135 @@ countdown(minutes = 5)
...
+
+
+## Monitor your model's outputs
+
+### Python
+
+```{python}
+#| fig-align: center
+#| output-location: slide 
+## TODO for Isabel: make plot like R one after this
```
Comment (juliasilge) [inline slides/04-monitor.qmd:256]: @isabelizimm can you fill in this chunk, to be like the one below for R?

File: slides/04-monitor.qmd [hunk 2]
```diff
@@ -194,6 +378,46 @@ countdown(minutes = 7)
...
+```{r}
+augment(v, inspections_new) |> 
+  group_by(facility_type) |> 
+  sensitivity(results, .pred_class)
+```
+
+### Python
+
+```{python}
+## TODO for Isabel: compute sensitivity by facility type for monitoring data
```
Comment (juliasilge) [inline slides/04-monitor.qmd:396]: @isabelizimm can you fill in this chunk, to be like the one above for R?

Comment (isabelizimm) [reply slides/04-monitor.qmd]: done

**Comment by isabelizimm** [comment]
This is ready to go! I was able to run this locally 🎉

**Comment by juliasilge** [comment]
Thank you so much!

**Pull request merged**